opencog/scm/logger-utils.scm

```diff
@@ -0,0 +1,35 @@
+;
+; logger-utils.scm
+;
+;;; Commentary:
+;
+; Helper to use ice-9 format in logger functions
+;
+;;; Code:
+; Copyright (c) 2015, OpenCog Foundation
+;
+
+(use-modules (opencog))
+(use-modules (ice-9 format))
+
+(define (cog-logger-error msg . args)
+  (cog-logger-error-str (apply format #f msg args)))
+
+(define (cog-logger-warn msg . args)
+  (cog-logger-warn-str (apply format #f msg args)))
+
+(define (cog-logger-info msg . args)
+  (cog-logger-info-str (apply format #f msg args)))
+
+(define (cog-logger-debug msg . args)
+  (cog-logger-debug-str (apply format #f msg args)))
+
+(define (cog-logger-fine msg . args)
+  (cog-logger-fine-str (apply format #f msg args)))
+
+(define (export-logger-utils)
+  (export cog-logger-error
+          cog-logger-warn
+          cog-logger-info
+          cog-logger-debug
+          cog-logger-fine))
```

Comment:
No particularly strong reasons, other than its pointless to have too many itty bitty files.  If its a module, put it in the module. there is no plausible scenario where a user would try to load that file stand-alone without having previously loaded the module.